dataprofiler/tests/test_utils_global.py

```diff
@@ -0,0 +1,37 @@
+"""Validates that generator intakes DATAPROFILER_SEED properly."""
+import unittest
+
+import numpy as np
+
+import dataprofiler.utils_global as utils_global
+
+
+class TestGetRandomNumberGenerator(unittest.TestCase):
```

Comment:
the test really should be testing all the states of the get_random_number_generator function
- 1) test returning the first `rng`
- 2) test `DATAPROFILER_SEED` ==  True and settings._seed == False
- 3) test `DATAPROFILER_SEED` ==  False and settings._seed == True
- 4) test `DATAPROFILER_SEED` ==  True and settings._seed == True
- 5) test `DATAPROFILER_SEED` ==  False and settings._seed == False
- 6) test for entering try / except, and return the try
- 7) test for entering try/except, and raise the exception --> validate with `assertRaisesRegex()`

Reply:
currently the test is really just testing `rng` and that is returns proper things but the test needs to cover the function and all the states / potential states of the function itself and not the object `rng`

Reply:
@jacob-buehler all these are covered? Haven't had a chance to review the PR yet in detail